packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -216,6 +280,14 @@ export default {
         }
       },
       immediate: true
+    },
+    space: {
+      handler: function () {
+        if (this.showSpaceMembers) {
```

Comment:
Should only be loaded if the space has changed, right?

Reply:
Please have a look again, I moved the `loadSpaceTask` to the `FileShares` component. IMO there is no need for that to be fetched in the Sidebar component. It just adds more complexity.